ll.tm

```diff
@@ -2639,10 +2639,10 @@ FCmpInst -> FCmpInst
 
 # ref: ParsePHI
 #
-#   ::= 'phi' Type '[' Value ',' Value ']' (',' '[' Value ',' Value ']')*
+#   ::= 'phi' FastMathFlag* Type '[' Value ',' Value ']' (',' '[' Value ',' Value ']')*
 
 PhiInst -> PhiInst
-	: 'phi' Typ=Type Incs=(Inc separator ',')+ Metadata=(',' MetadataAttachment)+?
+	: 'phi' FastMathFlags=FastMathFlag+? Typ=Type Incs=(Inc separator ',')+ Metadata=(',' MetadataAttachment)+?
```

Comment:
For consistency with the other instructions using fast math flags, can we use `*` instead of `+?` as both mean 0 or more.

```suggestion
	: 'phi' FastMathFlags=FastMathFlag* Typ=Type Incs=(Inc separator ',')+ Metadata=(',' MetadataAttachment)+?
```

e.g. this is the grammar of the recently added `fneg` instruction:

```
FNegInst -> FNegInst
	: 'fneg' FastMathFlags=FastMathFlag* X=TypeValue Metadata=(',' MetadataAttachment)+?
;
```